discojs/discojs-core/src/default_tasks/wikitext.ts

```diff
@@ -0,0 +1,40 @@
+import type { Model, Task, TaskProvider } from '..'
+import { TrainingSchemes, models } from '..'
+
+export const wikitext: TaskProvider = {
+  getTask (): Task {
+    return {
+      id: 'wikitext-103',
+      displayInformation: {
+        taskTitle: 'Wikitext 103 Raw',
+        summary: {
+          preview: 'In this challenge, we ask you to do next word prediction on a dataset of Wikipedia articles.',
+          overview: 'Wikitext-103-raw is a dataset comprising unprocessed text excerpts from Wikipedia articles, designed for tasks related to natural language processing and language modeling.'
```

Comment:
add a comment what tokenizer is used (type, and pretrained tokenizer downloaded from ...)

Reply:
and also a comment on what is to expect (like if you train alone for 5mins or 5h, you should expect this train/test loss value roughly, and that the resulting model will somehow sound english?)

Reply:
I think running gpt-micro on Shakespeare for 1 epoch should already produce some "shakespearesque" output